gen_cert.py

```diff
@@ -173,6 +173,13 @@ def interstitial_factory():
         # pdfs differently for the different templates
         self.template_version = cert_data.get('VERSION', 1)
         self.template_type = 'honor'
+        # Default Font
+        self.template_font_name = 'OpenSans'
+        self.template_font_type = 'Light'
+        # Check for yaml template font
+        if cert_data.get('font'):
+            self.template_font_name = cert_data.get('font')['name']
+            self.template_font_type = cert_data.get('font')['type']
```

Comment:
Never use `dict.get('key1')['key2']`.

```python
cert_data.get('font')['name']
```

is sort of pointless. `dict.get` is only useful when you're uncertain if the key exists in the dictionary (and may want to provide a fallback/default).
But here, since you call `get` _but_ then immediately index into it (pulling the name and type keys), there's no benefit to calling `get` because if the first key doesn't exist (`font`), then indexing `name` and `type` will break.

In this case, you are (sort of) checking if font exists in the dict, but the proper idiom is:

```
if key in collection:
    pass

if 'font' in cert_data:
   pass
```

I like @caesar2164 's approach best though.

NB: I think the underlying implementation of `dict.get` is actually:

```python
def get(dictionary, key, default=None):
    if key in dictionary:
        return dictionary[key]
    else:
        return default
```